docs/src/using_jupyter/emacs-org-mode.md

```diff
@@ -0,0 +1,10 @@
+Emacs Org Mode (using Babel) is a package for the Emacs text editor that connects with Jupyter kernels, such as stata_kernel, 
+to display results interactively inside the text editor.  Emacs also allows full console access to jupyter kernels 
+for an interactive Stata experience. Like jupyter notebook or lab, it allows for mixing of text narrative, latex math, 
+with code and Stata results.  Emacs Org Mode allows for fine grained control of document output for producing pdf manuscripts.
+
+Installation instructions can be found [here](https://rlhick.people.wm.edu/posts/stata_kernel_emacs.html).
```

Comment:
It might be worth it to copy some of that post into the documentation here. Similar to StackOverflow not allowing posts that are just stubs to links elsewhere https://meta.stackexchange.com/a/8259.

Maybe at least the installation instructions?

Reply:
Will do.  Do I have to do a new pull request with those changes?

Reply:
Never mind.  Figured it out.  The instructions are included now.

Reply:
Also, I didn't add any info on the github mainpage `readme.md`.  I am fine if you decide to add info there or not.